packages/alias/README.md

```diff
@@ -140,9 +140,9 @@ Example:
 ```javascript
 // rollup.config.js
 import alias from '@rollup/plugin-alias';
-import resolve from '@rollup/plugin-node-resolve';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
```

Comment:
let's keep this scoped to node-resolve for now, and update the plugins individually. we'll want to do a patch release for each so the docs on npm match the repo. for that we need individual commits for each plugin to keep the changelogs and publishing automation correct.

Reply:
@NotWoods ping. can we do one PR per plugin or a separate commit per plugin? With the path forward in #360 I'm thinking that a separate PR might be easier to digest.

Reply:
I'll revert these doc changes since the path forward should allow us to keep the same import format.

Reply:
Awesome